src/electron/views/automated-checks/components/header-section.tsx

```diff
@@ -2,16 +2,16 @@
 // Licensed under the MIT License.
 import { NamedFC } from 'common/react/named-fc';
 import * as React from 'react';
-import { h1, text } from './header-section.scss';
+import * as styles from './header-section.scss';
```

Comment:
Any reason we can't import the styles like before from this instead of importing *?

Reply:
I've been trying out using `styles` as a consistent identifier for css classes after seeing it as a common pattern in a lot of the videos/blogs/etc I watched during day of learning; I think it makes it a little more readable during use for them to have a common prefix since they often have names that are otherwise easy to confuse with the classes/variables in use for the implementation of the class

Reply:
Gotcha! I see what you mean; maybe bring this up in engineering discussion as well?